### DISCUSSION TITLE
How do I find an entry for an attachment?

### DISCUSSION THREAD

**Discussion by timkgh** [dani-garcia/vaultwarden#5326] [Q&A] [answered]
On the filesystem on the server side in `attachments` I can see an older attachment that I think the entry for is deleted. Not sure what's in that attachment as it's encrypted. How can I find the entry based on the attachment ID or its folder ID? Or how can I decrypt the file?
Thank you.

**Comment by stefan0xC** [comment]
The file should be in a folder named after the cipher uuid.
https://github.com/dani-garcia/vaultwarden/blob/ed4ad67e732c213beaec78970cdb68e48bee3dc1/src/db/models/attachment.rs#L34-L35

**Comment by timkgh** [comment, marked as answer]
Perfect, thanks! For future readers you can search in the web UI for something like `>shortid:38ed839f` (exact syntax) when the attachment folder name is `38ed839f-a963-...`
